src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Home/Index.cshtml

```diff
@@ -235,13 +257,29 @@
                             {
                                 <div class="border prize-wrapper flex-column flex-fullheight">
                                     <div class="area_title text-center mb-30">
-                                        <h3>@string.Format(CultureInfo.InvariantCulture, @Week.ToUpper(),  (i +1))</h3>
+                                        <h3>@string.Format(CultureInfo.InvariantCulture, @Week.ToUpper(), (i + 1))</h3>
                                     </div>
                                     @if (i == 0)
                                     {
                                         <div class="flex-centered flex-grow">
-                                            <a href="https://www.objectivity.co.uk/" target="_blank">
-                                                <img src="~/theme-2019/images/partners/objectivity.jpg" alt="objectivity" title="Objectivity" class="img-responsive center-block" />
+                                            <a href="https://career.softserveinc.com/en-us/poland" target="_blank">
+                                                <img src="~/theme-2019/images/partners/softserve.jpg" alt="softserve" title="SoftServe" class="img-responsive center-block" />
+                                            </a>
+                                        </div>
+                                    }
+                                    @if (i == 1)
```

Comment:
use _else_ - it will be more efficient (we will not check 3 times for every partner)

Reply:
Good Point, however its not possilbe to use 'else if' or 'else' in that particular case without changing code structure.

![obraz](https://user-images.githubusercontent.com/27915290/99990404-0e2b1600-2db4-11eb-84d2-647faccffc96.png)

In my opinion its easier now.

Reply:
else without @ https://www.w3schools.com/asp/razor_cs_logic.asp

Reply:
you are right. Sorry, totally forgot check without '@'